### DISCUSSION TITLE
Linking tasks to journal entries

### DISCUSSION THREAD

**Discussion by DickinsonSa** [TechbeeAT/jtxBoard#339] [Q&A]
I note in the app description it states "e.g. create meeting minutes and link your tasks to them".

What's the current thinking on linking - is it possible? Is it scheduled for the next major version? Is it implemented but not for journals - yet, or in the future? 

If it's here now, how's it done?

Sorry for the plethora of questions.

Thanks.

**Comment by patrickunterwegs** [comment]
Hey @DickinsonSa , 
Basically you can already go to the detail view of any entry and create subtasks or linked notes for it (e.g. through the top menu). Those subtasks and linked notes are independent entries that have a link to a parent.

In jtx Board 2.3.1 or 2.3.2 there will be the possibility to link any parent or child through the list view or the detail view!
